pages/common/tee.md

```diff
@@ -10,6 +10,10 @@
 
 `echo "example" | tee -a {{FILE}}`
 
+- Print standard input to the terminal, and also pipe it into another program for further processing:
+
+`echo "example" | tee /dev/tty | xargs printf "\033[1;34m%s\033[m\n"`
```

Comment:
/dev/tty in tokens `{{}}` please. Same for printf. Also, no need to colorize the output and make the example complicated. We want the pages to be simple and readable. printf "[%s]" is fine.

Reply:
Updated in c6bb56d651b0e78b966f1daf18c10010334f473f. I wrapped the entire last command in the tokens; I hope that’s correct, but I couldn’t quite figure out from the docs what the intended approach for tokens representing entire commands!